packages/components/src/badge/src/Badge.module.css

```diff
@@ -0,0 +1,90 @@
+.hop-Badge  {
+    /* Default */
+    --hop-Badge-font-family: var(--hop-overline-font-family);
+    --hop-Badge-font-size: var(--hop-overline-font-size);
+    --hop-Badge-font-weight: var(--hop-overline-font-weight);
+    --hop-Badge-letter-spacing: 0.015rem;
+    --hop-Badge-line-height: var(--hop-overline-line-height);
+    --hop-Badge-top-offset: var(--hop-overline-top-offset);
+    --hop-Badge-bottom-offset: var(--hop-overline-bottom-offset);
+    --hop-Badge-height: 1.5rem;
+    --hop-Badge-padding-inline: var(--hop-space-inset-sm);
+    --hop-Badge-border-radius: var(--hop-shape-pill);
+
+    /* Primary */
+    --hop-Badge-primary-text-color: var(--hop-primary-text-strong);
+    --hop-Badge-primary-background-color: var(--hop-primary-surface-strong);
+
+    /* Secondary */
+    --hop-Badge-secondary-text-color: var(--hop-neutral-text-weak);
+    --hop-Badge-secondary-background-color: var(--hop-neutral-surface-weak);
+
+    /* Disabled */
+    --hop-Badge-text-color-disabled: var(--hop-neutral-text-disabled);
+    --hop-Badge-background-color-disabled: var(--hop-neutral-surface-disabled);
+
+    /* Internal Variables */
+    --font-family: var(--hop-Badge-font-family);
+    --font-size: var(--hop-Badge-font-size);
+    --font-weight: var(--hop-Badge-font-weight);
+    --letter-spacing: var(--hop-Badge-letter-spacing);
+    --line-height: var(--hop-Badge-line-height);
+    --top-offset: var(--hop-Badge-top-offset);
+    --bottom-offset: var(--hop-Badge-bottom-offset);
+    --height: var(--hop-Badge-height);
+    --padding-inline: var(--hop-Badge-padding-inline);
+    --border-radius: var(--hop-Badge-border-radius);
+
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+
+    box-sizing: border-box;
+    inline-size: max-content;
+    block-size: var(--height);
+    padding-inline: var(--padding-inline);
+
+    font-family: var(--font-family);
+    font-size: var(--font-size);
+    font-feature-settings: "case";
+    font-weight: var(--font-weight);
+    line-height: var(--line-height);
```

Comment:
In  fact, merging the 2 previous comment with this one, should we create an "OverlineText" component, which handles font-feature, text-transform, fonts, pseudos alignements for the overline font. 

Then badges would be just using that overline text component, and it would simplify it's usage

Reply:
hm ya maybe we should do that, thanks